src/sentry/integrations/slack/threads/activity_notifications.py

```diff
@@ -187,9 +208,7 @@ def get_description(self) -> tuple[str, str | None, Mapping[str, Any]]:
             base_template = "an "
         else:
             base_template = "a "
-            # Make sure to make the proper noun have correct capitalization
-            # I.e. github -> Github, jira -> Jira
-            provider = provider.capitalize()
+            provider = external_issue.get_formatted_provider_name()
```

Comment:
you could clean up this code if you wanted too, but not a blocker
```
provider = external_issue.get_formated_ptovider()
if external_issue.get_provider() == external_issue.DEFAULT:
   ...
```
and this way you don't need to override the provider value